tests/function-controller/cmd/main.go

```diff
@@ -61,6 +74,8 @@ func loadConfig(prefix string) (config, error) {
 	return cfg, err
 }
 
-func failOnError(g *gomega.GomegaWithT, err error) {
-	g.Expect(err).NotTo(gomega.HaveOccurred())
+func failOnError(err error) {
+	if err != nil {
+		panic(err)
+	}
```

Comment:
why change it?

Reply:
I can log the error and then use `os.exit(1)`